qemu/tests/qemu_img_measure_convert_image.py

```diff
@@ -34,8 +35,14 @@ def _get_img_obj_and_params(tag):
 
     def _qemu_io(img, cmd):
         """Run qemu-io cmd to a given img."""
-        logging.info("Run qemu-io %s" % img.image_filename)
-        q = QemuIOSystem(test, params, img.image_filename)
+        image_filename = img.image_filename
+        logging.info("Run qemu-io %s" % image_filename)
+        if img.image_format == "luks":
+            image_secret_object = img._secret_objects[-1]
```

Comment:
We might miss something here: _secret_objects is protected...
But, as this is a workaround for QemuIOSystem(luks image), and we don't want to make code changes for QemuIOSystem, so I think it's OK.